src/components/SingleChoiceListItem.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from 'react'
+import { View } from 'react-native'
+import { capitalizeFirstLetter, getArticleColor } from '../utils/helpers'
+import { ARTICLES } from '../constants/data'
+import { COLORS } from '../constants/colors'
+import styled from 'styled-components/native'
+
+const StyledContainer = styled.View`
+  height: 55px;
+  margin-bottom: 5px;
+  border-radius: 2px;
+  border-width: 1px;
+  border-color: ${COLORS.lunesBlackUltralight};
+  border-style: solid;
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-items: center;
+`
+
+const StyledContainerClicked = styled(StyledContainer)`
+  background-color: ${COLORS.lunesBlack};
+  border-color: ${COLORS.lunesBlackUltralight};
+`
```

Comment:
Can we just merge these several StyledContainers into one. As it the article is send you:

color: ${props => props.correct ? "green" : "red"};

https://styled-components.com/docs/basics#adapting-based-on-props

Reply:
Sure!